packages/calcite-components/src/components/tooltip/tooltip.tsx

```diff
@@ -172,6 +171,14 @@ export class Tooltip implements FloatingUIComponent, OpenCloseComponent {
     }
   }
 
+  componentDidLoad(): void {
+    if (this.referenceElement && !this.effectiveReferenceElement) {
+      this.setUpReferenceElement();
+    }
+    connectFloatingUI(this, this.effectiveReferenceElement, this.el);
```

Comment:
` this.setUpReferenceElement();` already calls `connectFloatingUI` so no need here.